xml/CRSChannelCode.xml

```diff
@@ -2437,15 +2507,49 @@
         <codelist-item status="active" activation-date="2017-03-31">
             <code>61003</code>
             <name>
-                <narrative>Private investor in provider country</narrative>
-                <narrative xml:lang="fr">Investisseur privé dans le pays donneur</narrative>
+                <narrative>Investment funds and other collective investment institutions</narrative>
             </name>
         </codelist-item>
         <codelist-item status="active" activation-date="2017-03-31">
             <code>61004</code>
             <name>
-                <narrative>Other non-bank entity in provider country</narrative>
-                <narrative xml:lang="fr">Autre entité non-bancaire dans le pays donneur</narrative>
+                <narrative>Holding companies, trusts and Special Purpose Vehicles</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61005</code>
+            <name>
+                <narrative>Insurance Corporations</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61006</code>
+            <name>
+                <narrative>Pension Funds</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61007</code>
+            <name>
+                <narrative>Other financial corporations</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61008</code>
+            <name>
+                <narrative>Exporters</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61009</code>
+            <name>
+                <narrative>Other non-financial corporations</narrative>
+            </name>
+        </codelist-item>
+        <codelist-item status="active">
+            <code>61010</code>
+            <name>
+                <narrative>Retail investors</narrative>
             </name>
         </codelist-item>
```

Comment:
Codes `61005`, `61006`, `61007`, `61008`, `61009`, `61010` all added as part of July 2019 update